clients/js/test/adversarials.test.ts

```diff
@@ -323,6 +323,6 @@ test('a custom program cannot CPI into withdrawMarginAccountCpiTamm', async (t)
     (tx) => signAndSendTransaction(client, tx)
   );
 
-  // expect error to happen when awaiting the tx (signer privilege violation)
+  // Expect tx to fail with seed constraint violation
```

Comment:
Can you use `expectCustomError` from common.ts to ensure we get the expected error? The `test-helpers` package now has Tensor shared errors and Anchor errors so you can import the [seeds constraint error ](https://github.com/tensor-foundation/toolkit/blob/2b5e26b755d4be7e049188917b58e5c7a53bf0df/test-helpers/src/shared/errors.ts#L37)from there.